api/src/main/resources/client_api.yaml

```diff
@@ -208,6 +208,7 @@ definitions:
           The operator to use when comparing values. Defaults to EQUAL. If the "in" operator is used,
           either values or valueNumbers should be populated.
         $ref: "#/definitions/Operator"
+        default: '='
```

Comment:
So actually, I think there's something to be said for *not* defaulting stuff like this here... because it makes it impossible(?) for us to distinguish between researchers saying "yes give me equals" and not specifying an operator at all.

(I'm also changing the names of these operators.)

So maybe revert this change?

Reply:
Oh.. oops.. I was playing around with the default to see what happened and forgot to remove this. Please ignore this. I will revert.